### ISSUE TITLE
BALLView crashes when running into uncaught exceptions thrown by ringPerceptionProcessor

### ISSUE THREAD

**Issue by cschaerfe** [BALL-Project/ball#570]
1) When trying to render ball and stick representation of some (broken) PDB structures
BALLview crashes with the following error when trying to render a somewhat broken PDB structure (after downloading straight from PDB):
```
Finished downloading, please wait...

---------------------------------------------------
FATAL: uncaught exception!
---------------------------------------------------
last entry in the exception handler:
exception of type RingProcessorException occured in line 551 of ringPerceptionProcessor.C
error message:
Could not find a valid set of smallest rings for an input molecule.

Either the input molecule was not valid (e.g.: not a single connected component)
or the algorithm failed on this special topology.
---------------------------------------------------
```
An example for this behaviour is the PDB structure ```3BEH```.

This is most likely because the ball-and-stick rendering functionality (in atomBondModelBaseProcessor called by ballAndStickModel ?) lacks a try-catch block for output of the ringPerceptionProcessor. 

Notably, the line renderer works for the same protein without a problem.

**Comment by cschaerfe** [comment]
BALLView also crashes irreproducibly when running "Assign Bond Orders":

```
>   > assigning bond orders ...
The given parameter file does not exist! Using default!

---------------------------------------------------
FATAL: uncaught exception!
---------------------------------------------------
last entry in the exception handler:
exception of type RingProcessorException occured in line 551 of ringPerceptionProcessor.C
error message:
Could not find a valid set of smallest rings for an input molecule.

Either the input molecule was not valid (e.g.: not a single connected component)
or the algorithm failed on this special topology.
```

Exception occured for a molecule consisting of the first 5 residues in chain A of ```3beh```

**Comment by philthiel** [comment]
Fixed by [Commit](029fd2b9a0365e5926bcb4e2c227533efb356423) that catches this error.
